modules/environment/README.md

```diff
@@ -15,6 +15,7 @@ ZSH Options
 | AUTO_RESUME | Resume existing jobs before creating a new job |
 | LONG_LIST_JOBS | Use the verbose list format by default |
 | NOTIFY | Report job status immediately instead of waiting for new prompt |
+| INTERACTIVECOMMENTS | Recogniz `#` as comments
```

Comment:
@ericbn Thanks a lot, I have fixed the description just like you told me.
These suggestions are helpful for my English to write.